slicer-plugin/NvidiaAIAA/SegmentEditorNvidiaAIAALib/SegmentEditorEffect.py

```diff
@@ -149,17 +147,21 @@ def currentSegmentID(self):
 
     def updateServerSettings(self):
         self.logic.setServer(self.serverUrl())
-        self.logic.setUseCompression(slicer.util.settingsValue("NVIDIA-AIAA/compressData",
-                                                               True,
-                                                               converter=slicer.util.toBool))
+        self.logic.setUseCompression(slicer.util.settingsValue(
+            "NVIDIA-AIAA/compressData",
+            True, converter=slicer.util.toBool))
+        self.logic.setUseSession(slicer.util.settingsValue(
+            "NVIDIA-AIAA/aiaaSession",
+            True, converter=slicer.util.toBool))
```

Comment:
do we want default to be false here also?

Reply:
fixed